packages/mds-db/processors.ts

```diff
@@ -12,6 +12,13 @@ import schema from './schema'
 import { vals_sql, cols_sql, vals_list, logSql, SqlVals } from './sql-utils'
 import { getWriteableClient, makeReadOnlyQuery } from './client'
 
+export async function getVehicleType(device_id: UUID): Promise<VEHICLE_TYPE | null>{
+  const vals = new SqlVals()
+  const query = `SELECT type FROM devices WHERE device_id = ${vals.add(device_id)}`
+  const [queryResult]  = await makeReadOnlyQuery(query, vals)
+  return queryResult ? queryResult.type : null
+}
+
```

Comment:
Just curious why you didn't just use `readDevice` and grab the type.

Reply:
I suppose that'd work too, though with this method it's cleaner processor logic since we get just the column we're looking for. Also, if the column name ever changes we would need to track down changes in any code other than the cache.